src/Umbraco.Cms.Integrations.Automation.Zapier/Migrations/ZapierMigration.cs

```diff
@@ -62,13 +62,17 @@ public class ZapierSubscriptionHookTable
 
             /// <summary>
             /// Column stores two types of references:
-            /// 1. content type for content triggers
+            /// 1. content type alias for content triggers
             /// 2. form ID for form triggers
             /// </summary>
             [Column("EntityId")]
             [Index(IndexTypes.UniqueNonClustered, Name = "IX_ZapierSubscriptionHook_EntityId")]
             public string EntityId { get; set; }
 
+            [Column("Type")]
+            [Index(IndexTypes.UniqueNonClustered, Name = "IX_ZapierSubscriptionHook_Type", ForColumns = "EntityId,Type")]
```

Comment:
Now you've added this index, you should remove the one on `EntityId` (as that, theoretically, is no longer unique by itself).